tests/WE2E/machine_suites/coverage.cheyenne

```diff
@@ -3,6 +3,7 @@ grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot
 grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16
 grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_HRRR
 grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
+grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta
```

Comment:
The point of the coverage tests is that by running a portion of the suite on each platform, all the comprehensive tests will have run. Should this only be done on one platform?

Reply:
Other tests seem to exist for all the machines; e.g,
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR
...

Reply:
Yes, the test should only be added to a single platform's coverage test suite.  I'd recommend either Hera GNU, Derecho, Orion, or Hercules (machines with only 11 tests).

Reply:
@MichaelLueken thanks for confirming!

Reply:
Is there a reason we have a coverage.cheyenne file now that Cheyenne has been decommissioned?

Reply:
I am adamant that this more expensive ensemble test not be included as a test on every machine. I appreciate @MichaelLueken's suggestion above about choosing one of the platforms with a lighter load.